doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1656,29 +1656,35 @@ analysis on inductive or co-inductive objects (see :ref:`inductive-definitions`)
    with complex predicates as the induction principles generated by
    ``Function`` or ``Functional Scheme`` may be.
 
-.. tacv:: induction @term in @goal_occurrences
+.. tacv:: induction @term over {+ @ident}
 
-   This syntax is used for selecting which occurrences of :n:`@term` the
-   induction has to be carried on. The :n:`in @goal_occurrences` clause is an
-   occurrence clause whose syntax and behavior is described in
-   :ref:`occurences sets <occurencessets>`. If variables or hypotheses not
-   mentioning :n:`@term` in their type are listed in :n:`@goal_occurrences`,
-   those are generalized as well in the statement to prove.
+   This syntax is used for generalizing the induction hypothesis over
+   the list of hypotheses mentioned in the ``over`` clause. For instance:
 
    .. example::
 
       .. coqtop:: reset all
 
          Lemma comm x y : x + y = y + x.
-         induction y in x |-   *.
+         induction y over x.
          Show 2.
 
-.. tacv:: induction @term with @bindings_list as @disj_conj_intro_pattern using @term with @bindings_list in @goal_occurrences
+.. tacv:: induction @term in @goal_occurrences
+
+   This syntax is used for selecting which occurrences of :n:`@term` the
+   induction has to be carried on. The :n:`in @goal_occurrences` clause is an
+   occurrence clause whose syntax and behavior is described in
+   :ref:`occurences sets <occurencessets>`. If variables or hypotheses not
+   mentioning :n:`@term` in their type are listed in :n:`@goal_occurrences`,
+   those are generalized as well in the statement to prove, as if they
+   had been given with the ``over`` clause.
+
+.. tacv:: induction @term with @bindings_list as @disj_conj_intro_pattern using @term with @bindings_list in @goal_occurrences over {+ @ident}
 
-.. tacv:: einduction @term with @bindings_list as @disj_conj_intro_pattern using @term with @bindings_list in @goal_occurrences
+.. tacv:: einduction @term with @bindings_list as @disj_conj_intro_pattern using @term with @bindings_list in @goal_occurrences over {+ @ident}
```

Comment:
Since you are modifying this line, could you transform it like this:

```
.. tacv:: induction @term with @bindings_list as @disj_conj_intro_pattern using @term with @bindings_list in @goal_occurrences over {+ @ident}
          einduction @term with @bindings_list as @disj_conj_intro_pattern using @term with @bindings_list in @goal_occurrences over {+ @ident}
```
This is the form to use when two objects share a common description.
Could you also put some `{?` `}` delimiters around each of the optional clauses? Thanks!